tests/integration/classification_test.py

```diff
@@ -4,41 +4,50 @@
 
 from pyatlan.cache.classification_cache import ClassificationCache
 from pyatlan.client.atlan import AtlanClient
+from pyatlan.model.enums import AtlanClassificationColor, AtlanTypeCategory
 from pyatlan.model.typedef import ClassificationDef
 
-CLS_NAME = "psdk-ClassificationTest"
+CLS_NAME = "psdk-classification"
 
-# NOTE: Tests are intended to run in the order specified in this file,
-# to complete creation prior to testing prior to deletion.
 
+class ClassificationTest:
+    @staticmethod
+    def create_classification(client: AtlanClient, name: str) -> ClassificationDef:
```

Comment:
I'm not sure why this method is here. All the asserts in the method make it look like it's really a test.